src/core/serializedtypes.js

```diff
@@ -308,25 +311,25 @@ const STInt64 = exports.Int64 = new SerializedType({
       if (value < 0) {
         throw new Error('Negative value for unsigned Int64 is invalid.');
       }
-      bigNumObject = new SJCL_BN(value, 10);
+      bigNumObject = new BN(value, 10);
     } else if (isString(value)) {
       if (!isHexInt64String(value)) {
         throw new Error('Not a valid hex Int64.');
       }
-      bigNumObject = new SJCL_BN(value, 16);
-    } else if (value instanceof SJCL_BN) {
-      if (!value.greaterEquals(0)) {
+      bigNumObject = new BN(value, 16);
+    } else if (value instanceof BN) {
+      if (value.ucmp(new BN(0)) == - 1) {
```

Comment:
We need to use `ucmp` which compares the values as if they are both unsigned. We should probably make a BN(0) const somewhere.